### ISSUE TITLE
Sails.lower(done) wil generate and mocha error for done() called multiple times

### ISSUE THREAD

**Issue by calidion** [balderdashy/sails#3303]
code
```js
after(function(done) {
  // here you can clear fixtures, etc.
  Sails.lower(done);
});
```
will generate error:

```sh
  ․ sails : 0ms
  1) sails "after all" hook

  1 passing (7s)
  1 failing

  1) sails "after all" hook:
     Uncaught Error: done() called multiple times
      at Suite.<anonymous> (test/bootstrap.test.js:66:3)
      at Object.<anonymous> (test/bootstrap.test.js:11:1)
      at Array.forEach (native)
      at node.js:814:3
```

**Comment by calidion** [comment]
code:
```js
after(function(done) {
  // here you can clear fixtures, etc.
  Sails.lower();
  done();
});
```

will remove the error.

seems lower function invoke callback more than once.

**Comment by sailsbot** [comment]
Thanks for posting, @calidion.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by adborden** [comment]
I'm also running into this with `sails@0.11.4`.
